ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.19__periodics.yaml

```diff
@@ -203,6 +203,14 @@ tests:
     - env: OPENSTACK_CLOUD
       resource_type: openstack-vexxhost-quota-slice
     workflow: hypershift-openstack-aws-conformance
+- as: e2e-openstack-nested-conformance
+  minimum_interval: 240h
```

Comment:
for now let's just use a periodic job that doesn't run that often because it's resource consuming. We can iterate on that later but I think it's fine for now.

Reply:
it'll run every 10 days for now, let me know if you think it's not frequent enough.